modules/openapi-generator/src/main/resources/kotlin-client/jvm-common/infrastructure/KotlinxDateTimeInstantAdapter.kt.mustache

```diff
@@ -0,0 +1,56 @@
+package {{packageName}}.infrastructure
```

Comment:
@WonderCsabo First of all, thanks for creating this PR.
This is a nice contribution.
To keep everything consistent, could you please rename this file to InstantAdapter.kt.mustache?
Thanks

Reply:
@4brunu i intentionally used this name. `Instant` type also exists in Java 8 / ThreetenBP, so my intention was to use this specific name to avoid confusion with those date libraries. But i can rename it if you think that would be better.

Reply:
Yes, because if Java 8 Instant is used in the future, it should be added to this file.
 See https://github.com/OpenAPITools/openapi-generator/blob/3e45a05d00fe4364e06cb06dc89ad5564bbf02f8/modules/openapi-generator/src/main/resources/kotlin-client/jvm-common/infrastructure/OffsetDateTimeAdapter.kt.mustache for example, it contains the adapter code to moshi, gson and KotlinX Serialization, all in the same file.

Reply:
Okay, i see that logic. In that case in the future, the guards for kotlinx-datetime can be removed as well. I renamed the file!